### PR TITLE
Deploy config

### PR DESCRIPTION
**Pull request by nomadicoder**
Add CircleCI access key

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -46,7 +46,7 @@ jobs:
       - checkout
       - add_ssh_keys:
           fingerprints:
-            - "08:05:8e:1f:ff:2e:4d:59:60:3e:a2:d5:e3:e2:b8:68"
+            - "2c:e8:55:f8:43:47:17:d6:b3:0d:15:23:72:27:13:ad"
             - "2f:37:66:17:11:ae:f9:0f:9e:52:98:41:fe:d0:0c:78"
       - run:
           command: bash .circleci/deploy-prod.sh
```
Comment (bibliotechy) [inline .circleci/config.yml:52]: This still references `deploy-prod.sh`, which does not exist.

File: .circleci/deploy-qa.sh [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+# install playbook role requirements
+pipenv run ansible-galaxy install -r requirements.yml
+# setup vault password retrieval from travis envvar
+cp .circleci/.vault ~/.vault
+# setup vault password retrieval from travis envvar
+chmod +x ~/.vault
+
+# deploy to qa using ansible-playbook
+echo "Running: pipenv run ansible-playbook -i inventory/qa playbook.yml --vault-password-file=~/.vault"
+pipenv run ansible-playbook -i inventory/qa playbook.yml --vault-password-file=~/.vault
```
Comment (bibliotechy) [inline .circleci/deploy-qa.sh:21]: This references an inventory which does not yet exist in https://github.com/tulibraries/ansible-playbook-gencon50/tree/qa/inventory

File: .circleci/config.yml (hunk 1 above)
Comment (nomadicoder) [reply .circleci/config.yml]: Changed to QA deploy

File: .circleci/deploy-qa.sh (hunk 2 above)
Comment (nomadicoder) [reply .circleci/deploy-qa.sh:21]: Added QA inventory to ansible-playbook-gencon50

**Pull request merged**